### PR TITLE
Added html for section 4

### LINKED ISSUE #7: html section 4
We need to create the layout of the section 4

### PR DESCRIPTION
**Pull request by Talina24**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,7 +59,17 @@ <h2 class="care__title">ZEN Care</h2>
     </div>
   </section>
   <!-- Секция 4 -->
-  <section>Секция 4</section>
+  <section>
```
Comment (nagorapro) [inline index.html]: Need add class name "cashback" and id name "cashback"

**Review by nagorapro — COMMENTED** [review]
Need fix)

**Comment by Talina24** [comment]
fixed, kindly check)

**Review by nagorapro — APPROVED** [review]
Good

**Pull request merged**